_posts/2021-12-28-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,356 @@
+across worker nodes.
+* Reduced graceful shutdown time for worker nodes.
+
+## Everything else
+
+* Added support for ARM64 in the Trino Docker image.
+* Added cancelling a Trino query execution when JDBC statement is closed.
+* Added `clear` command to the Trino CLI to clear the screen.
+* Improved tab completion for the Trino CLI.
+
+## Trino Summit
+
+In 2021 we also enjoyed a successful inaugural Trino Summit, hosted by 
+Starburst, with well over 500 attendees. There were wonderful talks
+given at this event from companies like Netflix, LinkedIn, EA, Robinhood, 
+Doordash, Tabular, and Stream Native. If you missed this event, we have the 
+[recordings and slides available](https://www.starburst.io/resources/trino-summit/).
+
+As a teaser, the event started with Commander Bun Bun playing guitar to AC/DC's,
+"Back In Black".
+
+<iframe src="https://www.youtube.com/embed/c_qUp0SGeKE"  
+width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" 
+scrolling="no" style="border:1px solid #CCC; border-width:1px; 
+margin-bottom:5px; max-width: 100%;" allowfullscreen> 
+</iframe>
+
+## 2022 Roadmap: Project Tardigrade
+
+One of the interesting developments that came out of Trino Summit was a feature
+Trino co-creator, Martin, talked about in [the State of Trino presentation](https://www.starburst.io/resources/trino-summit/?wchannelid=2ug6mgs5ao&wmediaid=o264qw85dj).
+He discussed a prioritization in granular fault tolerance and performance in the
+core engine. While Trino has been proven to run batch analytics workloads at 
+scale, many have avoided long-running batch jobs in fear of a query failure. The
+fault-tolerance feature introduces a direction that the Trino project is taking 
+that will focus on adopting Trino as a long-running batch query engine.
+
+The granular fault-tolerance is being thoughtfully crafted to maintain the 
+speed that Trino has over other query engines in this space, while increasing
+the resiliency of queries. In other words, rather than when a query runs out of
+resources or fails for any other reason, a subset of the query (the task) is
+retried. To support this intermediate stage data is persisted to replicated RAM 
+or SSD.
+
+<a title="Schokraie E, Warnken U, Hotz-Wagenblatt A, Grohme MA, Hengherr S, et al. (2012), CC BY 2.5 &lt;https://creativecommons.org/licenses/by/2.5&gt;, via Wikimedia Commons" 
+ href="https://commons.wikimedia.org/wiki/File:SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png"><img width="512" 
+ alt="SEM image of Milnesium tardigradum in active state - journal.pone.0045682.g001-2" src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png/512px-SEM_image_of_Milnesium_tardigradum_in_active_state_-_journal.pone.0045682.g001-2.png">
+</a>
+
+This will be a larger undertaking of features to be introduced in the coming 
```

Comment:
The project to introduce granular fault-tolerance into Trino is called
Project Tardigrade. It is a focus for many contributors now, and we will introduce you to details in the coming months. The project is named after...